ui-admin/src/api/enrolleeSearch.ts

```diff
@@ -241,3 +233,31 @@ export const facetValuesFromString = (paramString: string, facets: Facet[]): Fac
   }
   return facetValues
 }
+
+/**
+ * Converts a facet value into a string for display in the form: "Facet Label: value"
```

Comment:
this is a presentation-oriented function, so I'd move it out of the `api` folder and into, e.g., SearchCriteriaView.tsx